Stages.md

```diff
@@ -1,5 +1,7 @@
 # WHATWG Stages
 
+**[See here](https://github.com/search?q=org%3Awhatwg+type%3Aissue+is%3Aopen+label%3A%22stage%3A+1%22%2C%22stage%3A+2%22%2C%22stage%3A+3%22&type=issues) for the current list of features using the Stages process and their current status.**
```

Comment:
I think we should word this similarly to how we link to anchor permanence issues in /working-mode:

> `See [all features using the Stages process] and their current states.`

I also wonder if we should restrict this to `is:issue`. People might well want to use this on a PR?

Reply:
Done!